docs/developer/migration-troubleshooting.md

```diff
@@ -0,0 +1,123 @@
+## Troubleshooting
+
+### Your toolkit
+For a general overview, read [this documentation](https://www.algotech.solutions/blog/python/django-migrations-and-how-to-manage-conflicts/)
+
+
+Some common commands:
+- docker-compose exec app bash -- gets you into bash
+- ./manage.py showmigrations -- shows the current migrations that are finished, all should have [x]
+- ./manage.py makemigrations -- makes the migration
+- ./manage.py showmigrations -- now you should see the new/updated migration with a [ ]
+- ./manage.py migrate [folder name here, ie registrar]
```

Comment:
**Comment (non-blocking):** Not sure if helpful, but we could add a little explanation beside.

`- ./manage.py migrate [folder name here, ie registrar] -- applies those changes to db for specific folder`

(If you're good with it, I can add this change to the PR)

Reply:
Sure, glad to have it!